examples/scripts/ct_astra_tv_admm.py

```diff
@@ -9,7 +9,7 @@
 =====================================
 
 This example demonstrates the use of class
-[admm.ADMM](../_autosummary/scico.admm.rst#scico.admm.ADMM) to solve a
+[admm.ADMM](../_autosummary/scico.optimize.admm.rst#scico.optimize.admm.ADMM) to solve a
```

Comment:
Maybe I missed it in the meeting: Is it not possible to make set `ADMM = admm.ADMM` in `scico.optimize.__init__.py`? (In order to the duplication of `.admm.ADMM`.)

Reply:
Ah, looking later on, I see
    
    from scico.optimize.admm import ADMM, LinearSubproblemSolver
    
which would become

    from scico.optimize import ADMM
    from scico.optimize.admm import LinearSubproblemSolver
    
which is maybe more confusing.

Reply:
Exactly. If one only needs `ADMM` then one can use `from scico.optimize import ADMM`, but importing from `scico.optimize.admm` is also supported to allow single-line imports.

Reply:
But good thing you commented on that specific change. Many API cross-references in the examples were broken.